StepDefinitions/TestSteps.cs

```diff
@@ -0,0 +1,131 @@
+using System;
+using TechTalk.SpecFlow;
+
+namespace TestSpecs {
+    [Binding]
+    class TestStpes
+    {
+        [Given(@"I am on the (.*) website")]
+        public void GivenIAmOnTheNLWebsite(string countryCode)
+        {
+            setCountryCode(countryCode);
+            Driver.Navigate().GoToUrl(baseURLCK);
+        }
+
+        [Given(@"All cookies are set")]
+        public void GivenAllCookiesAreSet()
+        {
+            Driver.Manage().Cookies.DeleteCookieNamed("PVH_COOKIES_GDPR");
+            Driver.Manage().Cookies.DeleteCookieNamed("PVH_COOKIES_GDPR_ANALYTICS");
+            Driver.Manage().Cookies.DeleteCookieNamed("PVH_COOKIES_GDPR_SOCIALMEDIA");
+            Driver.Manage().Cookies.DeleteCookieNamed("newsletterSeen");
+            Driver.Manage().Cookies.AddCookie(new Cookie("PVH_COOKIES_GDPR", "Accept"));
+            Driver.Manage().Cookies.AddCookie(new Cookie("PVH_COOKIES_GDPR_ANALYTICS", "Accept"));
+            Driver.Manage().Cookies.AddCookie(new Cookie("PVH_COOKIES_GDPR_SOCIALMEDIA", "Accept"));
+            Driver.Manage().Cookies.AddCookie(new Cookie("newsletterSeen", "true"));
+            Driver.Navigate().GoToUrl(baseURLCK);
+        }
+
+        [Given(@"I have one item in my shopping basket")]
```

Comment:
* Don't put `Given` here,  use `StepDefinition(regex)`,  because this step could be used not only in `Given`, but also in `Then`
* It should be parameterised, so you could accept a number as the count of products in the basket, such as `StepDefinition(@"I have (\d+) products? in my shopping basket")`